pkg/vsphere/compute/placement/ranked.go

```diff
@@ -42,25 +38,83 @@ func (r rankedHosts) Len() int           { return len(r) }
 func (r rankedHosts) Swap(i, j int)      { r[i], r[j] = r[j], r[i] }
 func (r rankedHosts) Less(i, j int) bool { return r[i].score > r[j].score }
 
+// WeightConfiguration holds user-provided weights for different host metrics. These weights are
+// used to determine a host ranking.
+type WeightConfiguration struct {
```

Comment:
You may want to define a DefaultWeights instance of this that can be used - given I doubt `0` is a useful default value.

Reply:
I've currently implemented two versions of `NewRankedHostPolicy`, one that takes one of these and one that doesn't. If you use the one that doesn't, it creates a `WeightedConfiguration` instance using the defined defaults, but I agree - maybe it would be cleaner to just define a default struct here instead of a separate file with constants.